install-tools.sh

```
@@ -0,0 +1,183 @@
+done
+
+is_cmd() { command -v "$1" >/dev/null 2>&1; }
+
+can_sudo() { command -v sudo >/dev/null 2>&1; }
+has_apt() { command -v apt-get >/dev/null 2>&1; }
+
+run_cmd() {
+  if [[ "$DRY_RUN" == "1" ]]; then
+    echo "[DRY_RUN] $*"
+  else
+    eval "$@"
```

Shellcheck (0.10.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)